CoffeeChat/Controller/NetworkManager.swift

```diff
@@ -20,23 +20,33 @@ class NetworkManager {
     func pingServer() -> Future<Response<String>> {
         networking(Endpoint.pingServer()).decode()
     }
+    
+    func refreshUserSession(token: String) -> Future<Response<UserSession>> {
+        return networking(Endpoint.refreshUserToken(token: token)).decode()
```

Comment:
delete return, and also do same for `updateUserInterests` and `updateUserOrganizations`